### ISSUE TITLE
ARM64 builds are packaged with an x86_64 executable

### ISSUE THREAD

**Issue by Jontte** [timescale/promscale#582]
I'm trying to set up promscale on an arm64 platform. However it looks like arm64 docker images are shipped with an x86_64 promscale executable. Running the docker image results in exec format errors and the executable indeed seems to have the wrong arch:

```
$ docker run --rm -it \
    --entrypoint "/bin/cat" \
    timescale/promscale:0.3.0-arm64 \
    promscale \
    > promscale_executable

$ file promscale_executable 
promscale_executable: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, stripped
```

**Comment by Harkishen-Singh** [comment]
Thank you for reporting this.

Cc @cevian @VineethReddy02

**Comment by cevian** [comment]
@Jontte This should be fixed in #586 and `timescale/promscale:0.3.0-arm64` should also be fixed. Please let us know

**Comment by Jontte** [comment]
I can confirm this is now fixed. Many thanks for the prompt action!
